build.sbt

```diff
@@ -12,22 +14,31 @@ lazy val root = Project(id = "scalacache",base = file("."))
   .settings(commonSettings: _*)
   .settings(sonatypeSettings: _*)
   .settings(publishArtifact := false)
-  .aggregate(core, guava, memcached, ehcache, redis, caffeine)
+  .aggregate(coreRoot, guava, memcached, ehcache, redis, caffeine)
+
+lazy val coreRoot = Project(id = "scalacache-core", base = file("."))
```

Comment:
I don't think we want this `coreRoot` project. You end up with two projects in the same directory (`"."`), which confuses sbt.

The root project should aggregate both `coreJVM` and `coreJS`, making the `coreRoot` project unnecessary.